styleguide/derek/view-templates/interactive-map/interactive-map.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Zoolander Interactive Map Plugin', () => {
+  beforeEach(() => {
+    $.fn.rsInteractiveMap();
+  });
+
+  it('map is defined', () => {
+    expect($.fn.rsInteractiveMap).to.not.be.undefined;
+    expect($.fn.rsInteractiveMap.test.setLeftCoords).to.not.be.undefined;
+    expect($.fn.rsInteractiveMap.test.setTopCoords).to.not.be.undefined;
+  });
+
+  describe('calculate left value', () => {
+    it('should subtract half of circle width & tip width and return negative value', () => {
+      const data = {
+        w: 350,
+        c: 28,
+      };
+      expect($.fn.rsInteractiveMap.test.setLeftCoords(data))
+            .to.eql(-((data.w / 2) - (data.c / 2)));
+    });
+  });
+
+  describe('calculate left value off screen', () => {
+    it('should add 15 to the circle width', () => {
+      const data = {
+        offScreen: true,
+        c: 28,
+      };
+      expect($.fn.rsInteractiveMap.test.setLeftCoords(data))
+            .to.eql(data.c + 15);
+    });
+  });
```

Comment:
Since this is `setLeftCoords()`, which is also being tested in the test prior to this, I'd place this `it()` statement inside the previous `describe()` block and then modify that describe block to something like `Test setLeftCoords()`. 

`describe()` is a good way to lump similar tests together.  I'd do the same for `setTopCoords()` tests below.